build.sbt

```diff
@@ -99,7 +103,43 @@ lazy val junitDriver = module("junit-driver", "driver/bindings/junit")
   )
 
 lazy val scalaTests = testModule("scala-tests", "extensions/scala/tests")
-  .dependsOn(junitDriver)
+  .dependsOn(junitDriver, scalaProbeDriver)
+  .usesIdeaPlugin(scalaProbePlugin)
+
+
+lazy val examples = testModule("examples", "examples")
```

Comment:
Would be nice to reorder projects in build.sbt according to how they depend on each other. 
probe modules
scala-probe modules
examples

and within probes: api, plugin, driver, tests

Reply:
done